worker.js

```diff
@@ -0,0 +1,56 @@
+var Sway = require('sway')
```

Comment:
Yeah, I tested it with swagger-editor (does that count?)
webpack has a UMD-like check for Worker environments, something like...
```js
if(typeof window !== 'undefined'){
...
else if (typeof self !== 'undefined') { // Web Worker
...
```